src/BakerMemoryManager.cpp

```diff
@@ -199,9 +233,17 @@ void* BakerMemoryManager::allocate(std::size_t requestedSize, bool* gcOccured /*
 
         if (gcOccured && !*gcOccured)
             m_allocationsCount++;
```

Comment:
WTF? `m_memoryInfo.allocationsCount++` vs `m_allocationsCount++`. No need to double the info.